tests/unit_tests/bittensor_tests/test_wandb.py

```diff
@@ -27,5 +27,7 @@ def test_dendrite():
     del dendrite
 
 def test_metagraph():
-    metagraph = bittensor.metagraph()
-    metagraph.to_wandb()
\ No newline at end of file
+    config = bittensor.subtensor.config()
+    subtensor = bittensor.subtensor(network='nakamoto', config=config, )
```

Comment:
It's not good practice to connect to the production network during testing. Let's either change this to `Nobunaga` which is our test net or change use the mocked network.